environments/templates/_helpers.tpl

```diff
@@ -4,3 +4,13 @@
     {{- if $enabled }}@{{ $app }}{{ end }}
   {{- end }}
 {{- end }}
+
+{{- define "enabledServicesYamlList" -}}
+{{- $enabledServicesList := list -}}
+{{- range $app, $enabled := .Values.applications -}}
+  {{- if $enabled }}
+  {{- $enabledServicesList = append $enabledServicesList (quote $app) -}}
+  {{- end -}}
+{{- end -}}
+[ {{ join ", " $enabledServicesList }} ]
+{{- end -}}
```

Comment:
```suggestion
{{- define "enabledServicesYamlList" }}
{{- range $app, $enabled := .Values.applications }}
  {{- if $enabled }}
- {{ $app | quote }}
  {{- end }}
{{- end }}
{{- end -}}
```

Reply:
I usually try to avoid having to explicitly specify indent levels if I can, but I don't feel too strongly about it.